lntest/harness_assertion.go

```diff
@@ -2514,6 +2514,56 @@ func (h *HarnessTest) AssertClosingTxInMempool(cp *lnrpc.ChannelPoint,
 	return closeTx
 }
 
+// AssertLocalFCInsightsForClosedChannels asserts that the passed chanPoint
+// is found in the list of passed closedChannels and also asserts that the
+// local force close insights is as expected.
+func (h *HarnessTest) AssertLocalFCInsightsForClosedChannels(
+	channels []*lnrpc.ChannelCloseSummary,
+	channelPoint string, initiator string,
+	chainActions []string) {
+
+	passed := false
+	for _, channel := range channels {
+		if channel.ChannelPoint == channelPoint {
+			require.NotNil(h, channel.
+				LocalFirstCloseInsights)
+			require.Equal(h, channel.LocalFirstCloseInsights.
+				LocalForceCloseInitiator, initiator)
+			require.True(h, AssertChainAction(channel.
+				LocalFirstCloseInsights.
+				LocalForceCloseChainActions, chainActions))
+			require.NotZero(h, channel.LocalFirstCloseInsights.
+				LocalForceCloseBroadcastHeight)
+
+			passed = true
+
+			break
+		}
+	}
+
+	require.True(h, passed,
+		"Could not find channel with same chan point")
+}
+
+// AssertChainAction asserts that the passed chainActions is of expected
+// length and contains all the items in the actions slice as keys in the
+// chainActions map.
+func AssertChainAction(chainActions map[string]*lnrpc.ListHTLCHash,
```

Comment:
Please make this name more descriptive. It seems that it is asserting that the ChainActions match the supplied slice. Maybe call it `AsserChainActionsMatch`